react/Button/Button.less

```diff
@@ -111,3 +111,47 @@
 .fullWidth {
   width: 100%;
 }
+
+.group {
+  display: flex;
+  width: 100%;
+
+  .root {
+    overflow: visible;
+
+    & + .root {
+      margin-left: @grid-gutter-width / 2;
+    }
+
+    & + .root_isTransparent {
+      margin-left: 0;
+    }
+  }
+
+  .root_isTransparent {
+    padding-left: @grid-gutter-width;
+    padding-right: @grid-gutter-width;
+  }
+  @media only screen and (max-width: 960px) {
```

Comment:
Why we are not using our standard breakpoints here 🤔

Reply:
It's a hard one - I know that the css rules look all bit random but it's sorta just works best for the different sizes. This rule I can switch to our mixin but example the 500px below was picked as breaking this earlier the buttons looked to big, I have demo'ed this all with @stephaniemoss and she is happy with how it looks if it's any consideration?